csatar-plugins/csatar/controllers/JsonImport.php

```diff
@@ -73,13 +79,276 @@ public function __construct()
             'k'     => AgeGroup::where('association_id', $this->associationId)->where('name', 'Vándor')->first()->id ?? 0,
             'v'     => AgeGroup::where('association_id', $this->associationId)->where('name', 'Vegyes')->first()->id ?? 0,
         ];
+
+    }
+
+    private array $legalRelationshipMap = [];
+    private array $religionMap = [];
+    private array $allergiesMap = [];
+    private array $chronicIllnesMap = [];
+    private array $foodSensitivitiesMap = [];
+
+    private function prepareScoutRelatedMappings() {
+        $this->legalRelationshipMap = [
+            'a' => LegalRelationship::where('name', 'Alakuló csapat tag')->first()->id,
+            'ujonc' => LegalRelationship::where('name', 'Újonc')->first()->id,
+            'tag' => LegalRelationship::where('name', 'Tag')->first()->id,
+            'ttag' => LegalRelationship::where('name', 'Tiszteletbeli tag')->first()->id,
+            'ervenytelen' => LegalRelationship::where('name', 'Érvénytelen adat')->first()->id,
+        ];
+
+        $this->religionMap = [
+            'adv'  => Religion::where('name', 'Adventista')->first()->id,
+            'bapt' => Religion::where('name', 'Baptista')->first()->id,
+            'evan' => Religion::where('name', 'Evangélikus')->first()->id,
+            'gkat' => Religion::where('name', 'Görög katolikus')->first()->id,
+            'jeh'  => Religion::where('name', 'Jehova tanúi')->first()->id,
+            'kat'  => Religion::where('name', 'Római katolikus')->first()->id,
+            'mas'  => Religion::where('name', 'Más felekezethez tartozó')->first()->id,
+            'muz'  => Religion::where('name', 'Muzulmán')->first()->id,
+            'ort'  => Religion::where('name', 'Ortodox')->first()->id,
+            'ref'  => Religion::where('name', 'Református')->first()->id,
+            'unit' => Religion::where('name', 'Unitárius')->first()->id,
+        ];
+
+        $this->allergiesMap = [
+            "Nincs" => Allergy::where('name', 'Nincs')->first(),
+            "Darázscsípés" => Allergy::where('name', 'Rovarméreg allergia')->first(),
+            "Por, pollen, macskaszőr, árpa" => Allergy::where('name', 'Egyéb')->first(),
+            "az idiótákra" => Allergy::where('name', 'Nincs')->first(),
+            "por" => Allergy::where('name', 'Egyéb')->first(),
+            "-" => Allergy::where('name', 'Nincs')->first(),
+            "házi por, őszi mezei füvek, dió" => Allergy::where('name', 'Egyéb')->first(),
+            "Brufen, szalicil/ szalicilsav" => Allergy::where('name', 'Egyéb')->first(),
```

Comment:
Ez nem Gyogyszerallergia?

Reply:
A szalicil-t és szalicilsav-at használják ételekben is.